grakn-engine/src/test/java/ai/grakn/client/TaskClientTest.java

```diff
@@ -1,174 +0,0 @@
-/*
- * Grakn - A Distributed Semantic Database
- * Copyright (C) 2016  Grakn Labs Limited
- *
- * Grakn is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * Grakn is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
- *
- */
-
-package ai.grakn.client;
-
-import ai.grakn.engine.TaskId;
-import ai.grakn.engine.TaskStatus;
-import ai.grakn.engine.controller.SparkContext;
-import ai.grakn.engine.controller.TasksController;
-import ai.grakn.engine.tasks.manager.TaskManager;
-import ai.grakn.engine.tasks.manager.TaskSchedule;
-import ai.grakn.engine.tasks.manager.TaskState;
-import ai.grakn.engine.tasks.manager.TaskStateStorage;
-import ai.grakn.engine.tasks.mock.ShortExecutionMockTask;
-import ai.grakn.exception.GraknBackendException;
-import com.codahale.metrics.MetricRegistry;
-import junit.framework.TestCase;
-import mjson.Json;
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.core.IsEqual;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ExpectedException;
-import org.mockito.ArgumentMatchers;
-import org.mockito.Mockito;
-
-import java.time.Duration;
-import java.time.Instant;
-
-import static ai.grakn.engine.TaskStatus.CREATED;
-import static java.time.Instant.now;
-
-// TODO: move this test into grakn-client when possible
-public class TaskClientTest {
```

Comment:
Why is this test dieing?

Reply:
because I merged this PR like 10 times with master 👍 
I'll try and resurrect it